index.js

```diff
@@ -28,7 +28,16 @@ const exampleMovies = require("./movies");
       "James and the Giant Peach",
     ];
  */
-function getAllMovieTitles() {}
+function getAllMovieTitles(movies) {
+  if (!movies) {
```

Comment:
I like this falsy check but an empty object or an empty array would not actually trigger this.